newdle/auth.py

```diff
@@ -17,4 +17,8 @@ def login():
 
 @auth.route('/logout/')
 def logout():
-    return multipass.logout(url_for('index', _external=True), clear_session=True)
+    after_logout = url_for('index', _external=True)
```

Comment:
`after_logout_url` would be a better name here

Reply:
Done on https://github.com/indico/newdle/pull/220